fbchat/graphql.py

```diff
@@ -155,6 +214,7 @@ def graphql_to_user(user):
     if user.get('profile_picture') is None:
         user['profile_picture'] = {}
     c_info = get_customization_info(user)
+    event_reminders = [graphql_to_event(event) for event in user['event_reminders']['nodes']] if user.get('event_reminders') is not None else []
```

Comment:
The `user.get('event_reminders') is not None` can be replaced by simply `user.get('event_reminders')`. This is true in a few other places, too

Reply:
Done